### PR TITLE
принцип Архимеда

### PR DESCRIPTION
**Pull request by kao4uka**
added buoyant_force_from_volume.py/ buoyant_force_from_volume_test.py

### CODE REVIEW

File: symplyphysics/laws/hydro/buoyant_force_from_volume.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+from sympy import Eq, solve
```
Comment (blackyblack) [inline symplyphysics/laws/hydro/buoyant_force_from_volume.py:1]: Sorry but this law already exists: https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/dynamics/buoyant_force_from_density_and_volume.py

**Review by blackyblack — CHANGES_REQUESTED** [review]

**Comment by kao4uka** [comment]
oh, sorry, I didn’t notice, can I make another one?

**Comment by blackyblack** [comment]
> oh, sorry, I didn’t notice, can I make another one?

Of course. BTW it was very close to original implementation. It tells about high quality of your job.

**Comment by kao4uka** [comment]
> > oh, sorry, I didn’t notice, can I make another one?
> 
> Of course. BTW it was very close to original implementation. It tells about high quality of your job.

Thanks

**Pull request closed without merging**